src/fprime_gds/flask/static/js/vue-support/log.js

```diff
@@ -21,16 +21,19 @@ let template = `
         <div class="my-3">
             <v-select id="logselect"
                     :clearable="true" :searchable="true"
-                    :filterable="true"  :options="options"
+                    :filterable="true"  :options="logs"
                     v-model="selected">
             </v-select>
+            <input name="scroll" type="checkbox" v-model="scroll" />
+            <label for="scroll">Scroll Log Output</label>
         </div>
     </div>
     <div class="fp-scroll-container">
         <div class="fp-scrollable fp-color-logging">
             <pre><code>{{ text }}</code></pre>
         </div>
     </div>
+    <span>{{ error }}</span>
```

Comment:
If you're making changes before merging, could be worth having this be bold and/or red-ish to make it stand out a little